src/DataLib/fendl32B_retrofit/reaction_data.py

```diff
@@ -14,6 +14,68 @@
              'γ'     : array([ 0      ,      0      ])  # gamma emission
 }
 
+def count_emitted_particles(particle, emitted_particle_string):
+    """
+    Count emitted particles from a reaction given a target particle
+        and the particles produced in a neutron activation reaction.
+
+    Arguments:
+        particle (str): Name of the target particle produced in the reaction.
+            Options include n, p, alpha, d, t, and 3He, corresponding to
+            neutrons, protons, alpha particles, deuterons, tritons,
+            and helium-3 nuclides.
+        emitted_particle_string (str): Particle product(s) of the neutron
+            activation, of the format 'p' for a single proton for example or
+            '2n' for two neutrons etc.
+
+    Returns:
+        particle_count (int): Count of the target particle present in
+            the product.
+    """
+
+    num_str = ''
+    particle_count_index = emitted_particle_string.find(particle) - 1
+    if particle_count_index < 0:
+        particle_count = 0
+    else:
+        while emitted_particle_string[particle_count_index].isdigit():
```

Comment:
Good catch, but I think this should work?
```suggestion
    while particle_count_index >=0 and emitted_particle_string[particle_count_index].isdigit():
```

Reply:
Yeah, that's definitely nicer than how I had set it up. I'll double check that it works and throw it in.